spec/system/candidate_interface/candidate_can_see_their_previous_application_on_apply_again_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate can see previous application on apply again' do
+  scenario 'when a candidate visits their apply again application form there is a link to their previous application' do
+    given_apply_again_is_active
+    and_i_am_signed_in
+    and_i_have_an_unsuccessful_apply1_application
+
+    when_i_visit_my_application_complete_page
+    and_i_click_on_apply_again
+    and_i_click_on_start_now
+    and_i_click_on_the_link_to_my_previous_application
+    then_i_see_the_review_previous_application_page
+
+    when_i_click_back
+    then_i_see_my_current_application_page
+
+    when_i_try_and_visit_the_page_with_another_candidate_id
+    then_i_am_redirected_to_my_current_application_page
```

Comment:
Can we add these tests to the main Apply 2 feature test, to avoid having to do the setup twice?

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/candidate_interface/candidate_with_unsuccessful_application_applies_again_spec.rb

Reply:
Have done 👍